src/sdk/nsfs_glacier_backend/tapecloud.js

```diff
@@ -7,6 +7,8 @@ const { GlacierBackend } = require("./backend");
 const config = require('../../../config');
 const path = require("path");
 const { exec } = require('../../util/os_utils');
+const { spawn } = require("child_process");
+const { createInterface } = require("readline/promises");
```

Comment:
I would name this const readline = require(...) because and then use readline.createInterface because without the module name as context it is less clear.